src/read_vegparam.c

```diff
@@ -239,16 +239,23 @@ veg_con_struct *read_vegparam(FILE *vegparam,
         nrerror(ErrStr);
       }
 
+      for ( j = 0; j < 12; j++ ) {
```

Comment:
OK, I didn't realize that 4.2. was having constants replaced.  I'll do it.  Stay tuned...